policy-controller/grpc/src/lib.rs

```diff
@@ -320,5 +342,198 @@ fn to_authz(
         networks,
         labels,
         authentication: Some(authn),
+        metadata: None, // TODO fill
+    }
+}
+
+fn to_http_route(
+    name: &String,
+    route: &HttpRoute,
+    srv: &InboundServer,
+    cluster_networks: &[IpNet],
+) -> proto::HttpRoute {
+    let metadata = Metadata {
+        kind: Some(metadata::Kind::Resource(metadata::Resource {
+            group: "gateway.networking.k8s.io".to_string(),
+            kind: "HTTPRoute".to_string(),
+            name: name.to_owned(),
+        })),
+    };
+
+    let hosts = route
+        .hostnames
+        .iter()
+        .map(|hostname| match hostname {
+            Hostname::Exact(host) => linkerd2_proxy_api::http_route::HostMatch {
```

Comment:
this may be kind of a nitpick, but...what if we imported some of these names from `linkerd2_proxy_api`? at the very least, we might want to stick a `use linkerd2_proxy_api::{http_route, http_types}` in this file...